qutebrowser/browser/commands.py

```diff
@@ -838,6 +840,11 @@ def yank(self, what='url', sel=False, keep=False):
             sel: Use the primary selection instead of the clipboard.
             keep: Stay in visual mode after yanking the selection.
         """
+        self.yank_object = {
+            'sel': sel,
+            'what': what,
+            'keep': keep
+        }
```

Comment:
This seems like some "namespace pollution" here (why would a `CommandDispatcher` have a `yank_object`?), and I'd rather avoid that. It might also be prone to race conditions.

You should be able to simply pass those as arguments to `_yank_to_target`, or use [`functools.partial`](https://docs.python.org/3.6/library/functools.html#functools.partial) to pass them to `_yank_callback`.

Reply:
Hey yeah, will learn something new about python. Thanks for advice. )

Reply:
Ok, I decided to use inner functions. What you think?